src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java

```diff
@@ -120,6 +120,22 @@ public CustomResponse createInterest(JsonNode interestDetails) {
         JsonNode fetchedDemandJson = demandEntity.get().getData();
         ((ObjectNode) fetchedDemandJson).put(Constants.INTEREST_COUNT,
             fetchedDemandJson.get(Constants.INTEREST_COUNT).asInt() + 1);
+        Set<String> interestOrgSet = new HashSet<>();
+        if (fetchedDemandJson.has(Constants.INTEREST_ORG_SET) && !ObjectUtils.isEmpty(fetchedDemandJson.get(Constants.INTEREST_ORG_SET))) {
+          for (JsonNode node : fetchedDemandJson.get(Constants.INTEREST_ORG_SET)) {
+            interestOrgSet.add(node.asText());
+          }
+        }
+        String orgId = interestDetails.get(Constants.ORG_ID).asText();
```

Comment:
Just check null check

Reply:
orgId is a mandatory field so in the validation method itself we are checking whether it is empty , null or missing